### ISSUE TITLE
update meta.tests to include RuleName in the write-warning message

### ISSUE THREAD

**Issue by regedit32** [PowerShell/DscResource.Tests#276]
For each PSSA rule we loop through like this:
```
foreach ($errorPssaRuleOutput in $errorPssaRulesOutput)
{
    Write-Warning -Message "$($errorPssaRuleOutput.ScriptName) (Line $($errorPssaRuleOutput.Line)): $($errorPssaRuleOutput.Message)"
}
```

In that example, we could include `$($errorPssaRuleOutput.RuleName)` in the write-warning message, which would make for easier discovery when trying to find which rule documentation to look up or for suppressing by rule name.

**Comment by johlju** [comment]
Sounds good to me!

**Comment by johlju** [comment]
This was resolved in PR #275 I believe.
